Cloud/scaleway.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# <xbar.title>Scaleway Instances</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Nick Penkov</xbar.author>
+# <xbar.author.github>npenkov</xbar.author.github>
+# <xbar.desc>Let you start/stop scaleway server instances</xbar.desc>
+# <xbar.dependencies>scw,jq</xbar.dependencies>
+# <xbar.image>https://i.imgur.com/GgYNzMQ.png</xbar.image>
+
+# Dependencies:
+# [scw](https://github.com/scaleway/scaleway-cli)
+# [jq](https://stedolan.github.io/jq/)
+
+# Installation:
+# 1. Copy this script to your BitBar plugin folder
+# 2. Ensure the plugin file is executable by running chmod +x scaleway.20s.sh
+# 3. Change your SCW profile settings
+unset SCW_ACCESS_KEY SCW_SECRET_KEY SCW_DEFAULT_ORGANIZATION_ID SCW_DEFAULT_REGION SCW_DEFAULT_ZONE SCW_PROFILE
+SCW_CLI_PROFILE="default"
+ACTION="$1"
+INSTANCE="$2"
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+
+CMD_SCW=$(command -v /usr/local/bin/scw)
+CMD_JQ=$(command -v /usr/local/bin/jq)
```

Comment:
@npenkov 

Brew installs items into a different directory for M1 silicon. This could solve that:

```suggestion
BREWPATH=/usr/local/bin
if [[ $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
  BREWPATH=/opt/homebrew/bin
fi

CMD_SCW=$(command -v ${BREWPATH}/scw)
CMD_JQ=$(command -v ${BREWPATH}/jq)
```

What do you think?

Reply:
@sprak3000 Sounds good. Thanks!